### DISCUSSION TITLE
Is there an easy way to modify arguments?

### DISCUSSION THREAD

**Discussion by sanktanglia** [vfsfitvnm/frida-il2cpp-bridge#200] [General]
I need to change the implementation of a method such that i change the value of an argument being called. Is there a way to call the original implementation but with a modified argument?

**Comment by vfsfitvnm** [comment]
Of course there is: https://github.com/vfsfitvnm/frida-il2cpp-bridge/wiki/Snippets#methods

Just pass a different argument when it comes to calling the original implementation

**Comment by sanktanglia** [comment]
ahh thanks for that, it wasnt clear to me that invoking the method inside implementation calls the original version of the function
